otterdog/eclipse-jdtls.jsonnet

```diff
@@ -43,6 +43,7 @@ orgs.newOrg('eclipse-jdtls') {
       secret_scanning: "disabled",
       secret_scanning_push_protection: "disabled",
       web_commit_signoff_required: false,
+      allow_merge_commit: false,
```

Comment:
the property is already define a few lines above, if you just remove the line 37 it should be what you want as the default value for allow_merge_commit is already false.

Reply:
Thanks.